drivers/modem/gsm_ppp.c

```diff
@@ -430,6 +461,29 @@ static void gsm_finalize_connection(struct gsm_modem *gsm)
 	/* Attached, clear retry counter */
 	gsm->attach_retries = 0;
 
+	LOG_DBG("modem attach returned %d, %s", ret, "read RSSI");
+
+	/* Read connection quality (RSSI) */
+	LOG_INF("Read RSSI");
+	for (int i = 0; i < GSM_RSSI_RETRIES; i++) {
+		ret = modem_cmd_send_nolock(&gsm->context.iface,
+									&gsm->context.cmd_handler,
+									&read_rssi_cmd, 1,
+									"AT+CSQ",
+									&gsm->sem_response,
+									GSM_CMD_SETUP_TIMEOUT);
```

Comment:
Indentation, your tab size is probably not 8

Reply:
fixed, thank you for the hint